docs/running-tests/chrome-chromium-installation-detection.md

```diff
@@ -0,0 +1,100 @@
+# Installation and Detection Protocol of Browser Binaries and Webdrivers for Chrome and Chromium
+
+This is a detailed description of the process in which `wpt` detects
+and installs the browser components for Chrome and Chromium.
+This process can seem convoluted and difficult to understand at first glance,
+but the reason for this process is to best ensure these components are
+compatible with each other and are the intended items that the user
+is trying to test.
+## Chrome
+
+### Detection
+- **Browser Binary**
+: Because `wpt` does not offer installation of Chrome browser binaries,
+it will not attempt to detect a Chrome browser binary in the virtual environment
+directory. Instead, `wpt` attempts to check commonly-used installation locations on
+various operating systems. This detection process is only used if the user has not passed
+a binary path as an argument using the `--binary` flag.
+
+- **Webdriver**
+: ChromeDriver detection for Chrome will only occur if a valid browser binary
+has been found and the version for that binary is discernible. Once the browser
```

Comment:
In this context, it meant that we were able to successfully check the version of the binary. Of course, a "valid browser binary" means that we can do this, and so this statement is redundant. I am removing it to avoid confusion.